pallets/capacity/src/tests/mock.rs

```diff
@@ -81,7 +81,7 @@ impl pallet_balances::Config for Test {
 	type AccountStore = System;
 	type WeightInfo = ();
 	type FreezeIdentifier = RuntimeFreezeReason;
-	type MaxFreezes = ConstU32<1>;
+	type MaxFreezes = ConstU32<2>;
```

Comment:
This is still set to 1 in a lot of the mocks, although it doesn't matter for those I guess.

Reply:
I had originally set them all to 2, but I put it back for that reason.